pkg/collector/py/kubeutil.c

```diff
@@ -0,0 +1,21 @@
+// +build cpython,kubelet
+
+#include "kubeutil.h"
```

Comment:
Thanks for dedicating a C source module to the corresponding Python module, I'll move around some code after this PR is merged to make this practice more consistent.